prboom2/src/g_game.c

```diff
@@ -1630,7 +1630,7 @@ void G_DoCompleted (void)
   if (gamemapinfo)
   {
 	  const char *next = "";
-	  if (gamemapinfo->endpic[0] && (strcmp(gamemapinfo->endpic, "-") != 0) && gamemapinfo->nointermission)
+	  if (gamemapinfo->endpic[0] && (strcmp(gamemapinfo->endpic, "-") != 0) && !gamemapinfo->nointermission)
```

Comment:
Wait a minute, I just skimmed through this code yesterday. Why didn't I catch this?

This isn't directly related to the original problem in #286, right?

To me this reads like "if we have set an endpic and nointermission is not set, we should skip the intermission screen" which just doesn't sound right.

Reply:
Yes, you are right, I missed one important part of the Woof UMPAINFO implementation and got confused.